readthedocs/builds/automation_actions.py

```diff
@@ -62,3 +62,10 @@ def set_private_privacy_level(version, match_result, action_arg, *args, **kwargs
     """Sets the privacy_level of the version to private."""
     version.privacy_level = PRIVATE
     version.save()
+
+
+def delete_version(version, match_result, action_arg, *args, **kwargs):
+    """Delete a version if isn't marked as the default version."""
+    if version.project.default_version == version.slug:
+        return
```

Comment:
It would be good to `log.info` something here to know that this version is not deleted because it's the default.